data/korath/korath weapons.txt

```diff
@@ -381,28 +381,28 @@ outfit "Korath Repeater Turret"
 		"shield damage" 26
 		"hull damage" 22
 		"heat damage" 33
-	description "Korath repeaters fire compact bolts of superheated plasma at their targets. For every one of these turrets installed on a ship, an auxiliary cooling system is typically required to cope with its heat output."
+	description "Thermal Repeaters fire compact bolts of superheated plasma at their targets. For every one of these turrets installed on a ship, an auxiliary cooling system is typically required to cope with its heat output."
 
 
-outfit "Korath Piercer"
+outfit "Piercer Missile"
 	category "Ammunition"
 	cost 3500
 	thumbnail "outfit/piercer"
 	"mass" .3
 	"piercer capacity" -1
-	description "This missile is fired from a Korath Piercer Launcher."
+	description "This missile is fired from a Piercer Missile Launcher."
 
-outfit "Korath Piercer Rack"
+outfit "Piercer Missile Rack"
 	category "Ammunition"
 	cost 56000
-	thumbnail "outfit/korath piercer storage"
+	thumbnail "outfit/Piercer Missile storage"
```

Comment:
I don't see this image file being renamed in this PR.
I don't think the words are supposed to be capitalised, either.
~You just did a replace all, didn't you?~
![image](https://user-images.githubusercontent.com/20605679/186530053-fd56b22a-4539-49c3-a837-f264e61c10fb.png)

```suggestion
	thumbnail "outfit/korath piercer storage"
```
If you decide to rename the image file:
```suggestion
	thumbnail "outfit/piercer missile storage"
```
Though, note that the Piercer Missile image is just called `piercer` so this could be changed to `piercer storage`.
```suggestion
	thumbnail "outfit/piercer storage"
```

Reply:
You didn't expect me to do all these renames by hand, did you?

Thank God for text editors that do a find and replace across an entire directory tree
and for reviewers who catch when you forget a term in your regex